{{cookiecutter.repo_name}}/{{cookiecutter.package_name}}/apps/site/templatetags/site.py

```diff
@@ -209,3 +209,61 @@ def add_field_attributes(field, class_name, placeholder=True):
         'class': ' '.join((field.css_classes(), class_name)),
         'placeholder': field.label if placeholder else '',
     })
+
+
+@library.global_function
+@library.render_with('pagination/pagination.html')
+@jinja2.contextfunction
+def render_pagination(context, page_obj, offset=2, pagination_key=None):
+    '''Renders the pagination for the given page of items.'''
```

Comment:
As a docstring, this is incomplete; this comment should explain the truncation behaviour.